Update Powertoys docs

Jay-o-Way opened the pull request:
## All pages:

- https://github.com/microsoft/PowerToys/issues/12993 → text updates

## Specific pages:

- VCM → added image
- https://github.com/microsoft/PowerToys/issues/309 → added [!tip]
- https://github.com/microsoft/PowerToys/issues/9604 → added [!tip]
- https://github.com/microsoft/PowerToys/issues/1796 → added [!note]
- https://github.com/microsoft/PowerToys/issues/11363 → added [!important]
- https://github.com/microsoft/PowerToys/issues/13061 → text update (new image = #3442)
- https://github.com/microsoft/PowerToys/issues/14076#issuecomment-956034012 → added comment on VS Redistributable
- https://github.com/microsoft/PowerToys/issues/11842 → text updates (TO DO: nav menu \#help)

![Awake in menu](https://user-images.githubusercontent.com/65828559/139058650-ffa0299d-eb04-44f2-b03a-acdd7e573e55.png)

## Possible future additions?

- https://github.com/microsoft/PowerToys/issues/12711
- https://github.com/microsoft/PowerToys/issues/8440

FYI @crutkas @dedavis6797

Jay-o-Way committed:
Add note about special characters in file names

Adressing https://github.com/microsoft/PowerToys/issues/11363

Jay-o-Way committed:
"Awake" to "PowerToys Awake"

https://github.com/microsoft/PowerToys/issues/11842

Jay-o-Way committed:
Update color-picker.md

Added comment for non-mouse-over state (9604)
Adressed issue 12993 for this page

Jay-o-Way committed:
add comment on Explorer preview setting

Added PDF text.
Adressing https://github.com/microsoft/PowerToys/issues/1796 and one instance of issue 12993.

Jay-o-Way committed:
Update administrator.md

Small changes

Jay-o-Way committed:
Update fancyzones.md

Adressed 12993.

Jay-o-Way committed:
Update image-resizer.md

`title` to accomodate 12993

Jay-o-Way committed:
adressed 12993 and 11842 and added PDF text

Jay-o-Way committed:
Adressed issue 12993

Jay-o-Way committed:
Adressed 12993 and small tweaks

Also consolidated reserved (hot)keys

Jay-o-Way committed:
adressed 12993

Jay-o-Way committed:
Adressed 12993

Jay-o-Way committed:
12993

Jay-o-Way committed:
12993 (Windows 10) and 12749 (usage/source)

Jay-o-Way committed:
Merge branch 'docs' into update-docs

Jay-o-Way committed:
added VCM source image

update reference to image

crutkas commented on hub/powertoys/shortcut-guide.md:
```diff
@@ -13,15 +13,15 @@ This guide uses PowerToys to display common keyboard shortcuts that use the Wind
 ![Screenshot of shortcut overlay](../images/pt-shortcut-guide-large.png)
 
-Keyboard shortcuts using the Windows key <kbd>⊞ Win</kbd> can be used while the guide is displayed. The result of those shortcuts (active window moved, arrow shortcut behavior changes, etc) will be displayed in the guide.
+Keyboard shortcuts using the Windows key <kbd>⊞ Win</kbd> can be used while the guide is displayed. The result of those shortcuts (active window moved, arrow shortcut behavior changes et cetera) will be displayed in the guide.
```
revert this change.  , etc is standard phrase

crutkas reviewed (approved):
for 99%, this is fine

Jay-o-Way committed:
text tweaks

- add location of zones-settings.json
- "note" to "tip" and "note" to "important"
- "etc"

Jay-o-Way committed:
Merge branch 'MicrosoftDocs:docs' into update-docs

Jay-o-Way committed:
remove resolved comment

image is updated in #3442

Jay-o-Way committed:
add requirement for VS.Runtime 2015-2019

PRMerger12 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way committed:
Undo VS Runtime requirement text

PRMerger16 commented:
@Jay-o-Way : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way commented:
@mattwojo what's your opinion?

mattwojo commented:
> > image must be added to the repo
> 
> Done.
> 
> Question though: for the issue about PowerToys Awake naming, the text in the _menu_ would like to be updated too. I don't know or see how to get there. Is that something I can do or does that need special access?

Hey @Jay-o-Way -- Trying to make sure that I understand the issue here... so the intention is for the Awake feature to be officially called "PowerToys Awake" and it needs to be renamed as such in the TOC? (The left-side table of contents menu). The TOC menu file for PowerToys is located here: https://github.com/MicrosoftDocs/windows-uwp/blob/e45f5e769fad664fb3105283494e47282acf5ab5/hub/dev-environment/toc.yml
I'm happy to make that change.. just want to be sure I'm understanding correctly.

mattwojo commented on hub/powertoys/image-resizer.md:
```diff
@@ -60,7 +60,11 @@ For example, setting the filename format to: `%1 (%2)` on the file `example.png`
-You can also choose to retain the original *last modified* date on the resized image.
+<!-- [Jay] remove following warning if/when a solution is committed -->
+> [!IMPORTANT]
+> As mentioned above, you can use a backslash to create a new folder upon execution. Please be aware that, at this point - until a fix is committed, all special charaters are not filtered. For example: giving a preset a name with "1/4" in it, will also unintentionally create a folder. A colon `:` will result in corrupted files. For a complete list, see https://docs.microsoft.com/en-us/windows/win ...
```
We should make links like this one relative, so a more standard way to link this section is, that doesn't include a 'hardcoded' language-location in the URL (including the 'en-us' will force the link to show up in English, excluding it will allow the browser to choose based on the current language setting), so here's my recommended change:
 For a complete list, see [File and Directory Names](/windows/win32/fileio/naming-a-file#file-and-directory-names) in the WIN32 Local File Systems docs.

mattwojo committed:
Update link

PRMerger14 commented:
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

mattwojo reviewed (approved):
GREAT updates!! Thank you Jay!!!

mattwojo committed:
Merge branch 'docs' into update-docs

PRMerger20 commented:
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

Jay-o-Way commented:
I see you solved both your comments. Thanks. That's why we have you to check 🙂
